.travis.yml

```diff
@@ -53,14 +53,15 @@ install:
 script:
   - mkdir -p "$HOME/.ipython/profile_default"
   - "echo 'c.HistoryAccessor.enabled = False\n' > $HOME/.ipython/profile_default/ipython_config.py"
-  - export TEST_EXTRA=""
   - if [[ "$STATIC" == "false" ]]; then
-      if [[ "$COVERAGE" == "true" ]]; then
-        export TEST_EXTRA="--cov-config .coveragerc --cov nengo";
-      fi;
       python -c "import numpy; numpy.show_config()";
       python setup.py -q develop;
-      eval py.test nengo -n 2 -v --duration 20 "$TEST_EXTRA";
+      if [[ "$COVERAGE" == "true" ]]; then
+        coverage run --rcfile .coveragerc --source nengo -m py.test nengo -v --duration 20;
+        coverage report;
+      else
+        py.test nengo -n 2 -v --duration 20;
```

Comment:
We are duplicating the `-v` and `--duration 20` options here so they could get out of sync... but I think leave it as is for now; if we add more `py.test` options we should make an environment variable for them, but it's easier to read as it is here.